### ISSUE TITLE
Structures will not generate.

### ISSUE THREAD

**Issue by AsherJST** [stal111/Valhelsia-Structures#45]
This issue is basically the same as the other one (but a different version) where the person explained he had looked thousands of blocks in each direction and nothing will spawn. I am on java version 1.16.5 forge version 36.0.1 

I have tried to use the command /locate but it did not work and said that it couldnt find any nearby structures.

**Comment by EnterFor** [comment]
same....

**Comment by mrmyers365** [comment]
Hey! Try going to the config folders in .minecraft, and locate both client and server Valhesia Structures folders. If you check both, you will find that the client one is empty, while the server has all the codes you'll need for the mod to work properly. What I did was to delete the empty "client" folder, and then changed the name of the server one into "client". Then, I used the command /locate valhesia and teleport to the locations given and everything was perfect. This worked for me and I hope that it will work for you too!
